docs/custom-models.md

```diff
@@ -154,72 +51,33 @@ When anyone clones and opens the codebase the model was trained on, any models a
+#### Delete a user-associated model created within Visual Studio
 
-#### Delete a model created from the Azure DevOps task
+While your solution is open in Visual Studio, simply uncheck the checkbox accepting model training, on the View > Other Windows > IntelliCode page. The model will be deleted.
 
-1.	Delete the **Visual Studio IntelliCode Team Model Training** task from your pipeline.
-2.	Open Visual Studio and find the model ID by going to **View** > **Other Windows** > **List IntelliCode models**.
-      - You can also search for List IntelliCode models in Visual Studio Search (Ctrl + Q). 
-3.	You’ll be prompted to sign-in with the same account  you used to set up the IntelliCode service connection in Azure DevOps.
-4.	Copy the model ID for the model you’d like to delete.
-      - All IntelliCode models you have access to will be listed in the Output window. 
-5.	Go to **View** > **Other Windows** > **Delete IntelliCode models**. Paste the model ID into the box and hit **Delete**.
+#### Delete a repository-attached model created using Azure DevOps or a GitHub CI workflow
 
-When you delete your models, your request will remove the model from your account, and the data will be purged from IntelliCode’s systems within 72 hours.
+Simply remove the training task from your pipeline - the associated model will be removed within 30 days if not trained.
 
-## User-associated team models
-
-> [!NOTE] 
-> User-associated team completions models will be deprecated in the next year. We encourage you to utilize [repository-associated team models](#repository-associated-team-models) moving forward.
+## Machine-associated models
 
 ### Create your model
 
-> [!NOTE] 
-> Team completions is a preview feature for C# and is disabled by default. It can be enabled through Tools > Options > IntelliCode > C# team models for completions.
-
-Train a user-associated model:
+Train a machine-associated model:
 1.	Open the project or solution in Visual Studio.
-2.	Open the IntelliCode page by choosing View > Other Windows > IntelliCode Model Management.
-3.	Choose **Create new model**.
-
-### Share your model
-
-After you've trained a model, the **Share model** button appears. Click the button to copy the sharing link. From there, you can share the link with your collaborators.
-
- > [!NOTE]
- > Anyone who has the sharing link can access the model and its suggestions, so make sure that everyone who receives the link is aware of this.
-
-You can share your model with as many people as you like via the sharing feature. Team members can't retrain the model but they do see the same completion recommendations as you do.
-
-For more information about sharing completions models, see [How to: Share custom models](share-models.md).
+2.	Open the IntelliCode page by choosing **View** > **Other Windows** > **IntelliCode**
+3.	Review and accept the license terms checkbox at the bottom of the page. A machine-associated model will be trained automatically
```

Comment:
this only affects the current opened solution, these steps need to be repeated for others